rtgui/previewwindow.cc

```diff
@@ -199,6 +199,8 @@ bool PreviewWindow::on_motion_notify_event (GdkEventMotion* event)
 
         if (isMoving) {
             mainCropWin->remoteMove ((event->x - press_x) / zoom, (event->y - press_y) / zoom);
```

Comment:
Nope, it wasn't a bug but a change for the new mechanism, since it works incrementally now, it has to set the new 'press' position at each move.